src/client.js

```diff
@@ -1,3 +1,110 @@
-const Discord = require('discord.js')
+import { readdirSync } from 'fs'
+import { join } from 'path'
+import { Collection } from 'discord.js'
+import { CommandoClient } from 'discord.js-commando'
 
-module.exports = new Discord.Client()
+const {
+  OWNERS_IDS = '269617876036616193', // Default to @evan#9589
```

Comment:
Shouldn't it be `process.env.OWNERS_IDS` from `.env.example`

Reply:
Perhaps... but, well, the intent here was so that if the `.env` file is missing or if the `OWNERS_IDS` entry was missing from it that it wouldn't cause any errors.

It's just a fallback.

Reply:
```js
const {
  OWNERS_IDS = '269617876036616193', // Default to @evan#9589
  COMMAND_PREFIX = '!',
} = process.env
```
Is basically equivalent to this:

```js
let { OWNERS_IDS, COMMAND_PREFIX } = process.env
if (!OWNERS_IDS) OWNERS_IDS = '269617876036616193'
if (!COMMAND_PREFIX) COMMAND_PREFIX = '!'
```